src/ui/reducers/app.ts

```diff
@@ -21,12 +22,10 @@ import {
 } from "ui/state/app";
 import { getNonLoadingRegionTimeRanges } from "ui/utils/app";
 import { getSystemColorSchemePreference } from "ui/utils/environment";
-import { compareBigInt } from "ui/utils/helpers";
 import { prefs } from "ui/utils/prefs";
 import { isPointInRegions, isTimeInRegions, overlap } from "ui/utils/timeline";
 
 export const initialAppState: AppState = {
-  // analysisPoints: {},
```

Comment:
Delete?

Reply:
That _is_ a removal :) it was already commented out when I was looking at this file, so I just killed it.

Reply:
lol it's been a long day